tensorboard/components/tensor_widget/dtype-utils.ts

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+/**
+ * Determine if a data type is an integer type.
+ * @param dtype
+ * @return Whether the dtype is an integer type.
+ */
+export function isIntegerDType(dtype: string): boolean {
+  return (
+    dtype.match(/^int[0-9]+$/) != null || dtype.match(/^uint[0-9]+$/) != null
```

Comment:
prefer to use triple equals. I believe `match` doesn't return `undefined` so these `!=` can be `!==`, right?

Reply:
Done.